src/FE/FEsrc/pages/ProfileCreation.tsx

```diff
@@ -66,16 +63,16 @@ const ProfileCreationPage = (props) => {
 
     const genericErrorMessage = 'This field cannot be empty';
 
-    const addressValid = !isClinician ? (streetField.valid && cityField.valid && provinceField.valid && postalCodeField.valid) : (postalCodeField.valid);
+    const addressValid = postalCodeField.valid;
 
     const enableSubmit = nameField.valid && dateOfBirthField.valid && genderField.valid && addressValid;
 
```

Comment:
The address valid variable can be condensed into enable submit since it is just post code valid